serialization.rst

```diff
@@ -159,7 +159,8 @@ possible mapping values are:
 - ``any other string`` — defines the class name that the BSON array or BSON
   object should be deserialized at.
 
-  If the class implements the ``MongoDB\BSON\Unserializable`` interface, then
+  If the class implements the ``MongoDB\BSON\Unserializable`` interface, or
+  implicitly by implementing ``MongoDB\BSON\Persistable``, then
```

Comment:
I'd reword the clause as: "either directly or indirectly via ``MongoDB\BSON\Persistable``"